core/server/helpers/asset.js

```diff
@@ -2,13 +2,13 @@
 // Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`
 //
 // Returns the path to the specified asset. The ghost flag outputs the asset path for the Ghost admin
-var proxy = require('./proxy'),
+const proxy = require('./proxy'),
     _ = require('lodash'),
     getAssetUrl = proxy.metaData.getAssetUrl,
     SafeString = proxy.SafeString;
 
-module.exports = function asset(path, options) {
-    var hasMinFile = _.get(options, 'hash.hasMinFile');
+module.exports = (path, options) => {
```

Comment:
I think there are cases where arrow functions don't make sense 🤔In this case, it's a utility. We are loosing the power of named functions. I am not seeing a good reason to use arrow functions here?

Let me know if you have any input/opinion on that :)

Reply:
The only thing you gain with a named function is the ability to see errors in the stack trace down to a function level. Since the file is named asset.js I thought this would not cause an issue as we are only doing one thing in this file. . The stack will tell you that you have an error in file ./asset.js this should help boil down the error.

Reply:
https://www.linkedin.com/pulse/javascript-named-vs-anonymous-functions-chris-ng/

This article gives a decent breakdown.

Reply:
> The only thing you gain with a named function is the ability to see errors in the stack trace down to a function level

This can be very helpful sometimes.

`module.exports` is an object and if we are using arrow functions here, this results in

> console.log(module.exports)
> [Function]

This is an argument against using arrow functions here. Open for any more opinions.

From https://stackoverflow.com/questions/22939130/when-should-i-use-arrow-functions-in-ecmascript-6, it also describes that the access to `arguments` is different - the output is something different!

Reply:
Since the function in question does not use ```this``` inside of its self nor does it call ```this``` from outside of its self we will be fine to use an anonymous arrow function here. 

However, as a new contributor, I do not have a full scope of the project in mind yet. Until I can gain a better understanding of the project we can leave this as a named es5 function

Reply:
> If we're standardising es6 across our repos then for reference we only use arrow functions in Ghost-Admin when we want to preserve the this context within the function body.

Yeah 👍

Reply:
Sorry if I wasn't clear above. In Ghost-Admin we always use arrow functions in place of anonymous functions. However what this thread was talking about was named functions so I didn't clarify. `module.exports` is definitely a case for using named functions not an anonymous/arrow function.

Reply:
Although using arrow function in this situation is fine, I also prefer to not use it for clarity. If I was the author I would write function separately and export it as `module.exports = function name`.